### ISSUE TITLE
Doesn't load blob url with 1.2.7

### ISSUE THREAD

**Issue by smlee** [VadimDez/ng2-pdf-viewer#164]
I recently updated my package and was going nuts trying to figure out why none of the PDFs were loading. I rolled back to 1.2.6 and works fine. I don't know which dependency is causing this bug.

**Comment by VadimDez** [comment]
What error(s) did you get?

**Comment by DHFW** [comment]
I experienced this too. 1.2.6 works.

**Comment by smlee** [comment]
@VadimDez All the PDF that used to show up doesn't show up at all. No errors

**Comment by smlee** [comment]
1.2.6 shows up sometimes. Just had to downgrade to 1.2.5. Not sure if it's going to help but these are the packages I am using:
```JSON
  "dependencies": {
    "@angular/animations": "^4.4.3",
    "@angular/cdk": "^2.0.0-beta.11",
    "@angular/common": "^4.4.3",
    "@angular/compiler": "^4.4.3",
    "@angular/core": "^4.4.3",
    "@angular/flex-layout": "2.0.0-beta.8",
    "@angular/forms": "^4.4.3",
    "@angular/http": "^4.4.3",
    "@angular/material": "^2.0.0-beta.11",
    "@angular/platform-browser": "^4.4.3",
    "@angular/platform-browser-dynamic": "^4.4.3",
    "@angular/router": "^4.4.3",
    "angular2-jwt": "^0.2.3",
    "angular2-uuid": "^1.1.1",
    "auth0-js": "^8.10.1",
    "core-js": "^2.5.1",
    "hammerjs": "^2.0.8",
    "html2canvas": "^0.5.0-beta4",
    "jsonfn": "^0.31.0",
    "lodash": "^4.17.4",
    "moment": "^2.18.1",
    "ng2-pdf-viewer": "1.2.5",
    "ng2-responsive": "^0.8.4",
    "rxjs": "^5.4.3",
    "ts-helpers": "^1.1.2",
    "zone.js": "^0.8.17"
  },
  "devDependencies": {
    "@angular/cli": "^1.4.3",
    "@angular/compiler-cli": "^4.4.3",
    "@types/auth0-js": "^8.6.3",
    "@types/html2canvas": "^0.5.35",
    "@types/jasmine": "^2.6.0",
    "@types/lodash": "^4.14.76",
    "@types/node": "^8.0.31",
    "codelyzer": "^3.2.0",
    "jasmine-core": "^2.8.0",
    "jasmine-spec-reporter": "^4.2.1",
    "karma": "1.7.0",
    "karma-chrome-launcher": "^2.2.0",
    "karma-cli": "^1.0.1",
    "karma-jasmine": "^1.1.0",
    "karma-remap-istanbul": "^0.6.0",
    "protractor": "~5.1.2",
    "ts-node": "3.0.6",
    "tslint": "^5.7.0",
    "typescript": "~2.3.4"
  }
}```

**Comment by VadimDez** [comment]
fixed in 2.0.0
